src/imitation/scripts/config/train_preference_comparisons.py

```diff
@@ -32,6 +32,7 @@ def train_defaults():
 
     reward_trainer_kwargs = {
         "epochs": 3,
+        "weight_decay": 0.0,
```

Comment:
Out of curiosity: Have you already done experiments with non-zero weight decay, and has that helped? (Though I agree with setting this to zero by default unless we're pretty sure that some non-zero value consistently works better)

Reply:
Can omit this line, as it's already got a default value in `__init__`.